lib/svg-sprite.js

```diff
@@ -166,19 +166,18 @@ SVGSpriter.prototype._isVinylFile = function(file) {
  */
 SVGSpriter.prototype._transformShape = function(shape, cb) {
     const tasks = [];
-    const that = this;
-    const createTransformTask = function(transform) {
+    const createTransformTask = transform => {
         // If it's a custom transformer
         if (_.isFunction(transform[1])) {
```

Comment:
Ideally, I'd like to destructure transform[0] and transform[1] too, but maybe we leave this for later.